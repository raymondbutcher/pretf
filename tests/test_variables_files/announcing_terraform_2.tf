# https://www.hashicorp.com/blog/announcing-terraform-0-12#first-class-expression-syntax

variable "base_network_cidr" {
  default = "10.0.0.0/8"
}

resource "google_compute_network" "example" {
  name                    = "test-network"
  auto_create_subnetworks = false
}

resource "google_compute_subnetwork" "example" {
  count = 4

  name          = "test-subnetwork"
  ip_cidr_range = cidrsubnet(var.base_network_cidr, 4, count.index)
  region        = "us-central1"
  network       = google_compute_network.custom-test.self_link
}
