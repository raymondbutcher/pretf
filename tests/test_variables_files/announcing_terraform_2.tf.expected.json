[
  {
    "name": "base_network_cidr",
    "default": "10.0.0.0/8",
    "source": "announcing_terraform_2.tf"
  }
]